docs/release_notes/4.2.25-25.md

```diff
@@ -0,0 +1,31 @@
+# Percona Server for MongoDB 4.2.25-25 (2024-02-)
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 4.2.25-25 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for [MongoDB 4.2.25 Community Edition](https://www.mongodb.com/docs/manual/release-notes/4.2/#4.2.25---dec-6--2023).
+It supports MongoDB 4.2.25 protocols and drivers.
+
+## Release Highlights
+
+* Percona Server for MongoDB includes telemetry that fills in the gaps in our understanding of how you use Percona Server for MongoDB to improve our products. Participation in the anonymous program is optional. You can opt-out if you prefer not to share this information. [Read more about Telemetry](../telemetry.md).  
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* [SERVER-40459](https://jira.mongodb.org/browse/SERVER-40459) - Optimized the construction of the balancer's collection distribution status histogram
+
+* [SERVER-68434](https://jira.mongodb.org/browse/SERVER-68434) - Fixed the query planner logic to distinguish parameterized queries in the presence of a partial index that contains logical expressions (`$and`, `$or`).
+
+* [SERVER-71627](https://jira.mongodb.org/browse/SERVER-71627) - Improved performance of updating the routing table and prevented blocking client requests during refresh for clusters with 1 million of chunks.
+
+* [SERVER-80488](https://jira.mongodb.org/browse/SERVER-80488) - Avoided traversing routing table in balancer split chunk policy
+
+* [SERVER-81966](https://jira.mongodb.org/browse/SERVER-81966) - Fixed the issue that caused the modification of the original ChunkMap vector during the chunk migration and that could lead to data loss. The issue affects MongoDB versions 4.4.25, 5.0.21, 6.0.10 through 6.0.11 and 7.0.1 through 7.0.2. Requires stopping all chunk merge activities and restarting all the binaries in the cluster (both `mongod` and `mongos`). 
+
+Find the full list of bug fixes in the [MongoDB 4.2.25 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/4.2/#4.2.25---dec-6--2023)
+
+## New Features
+
+* [PSMDB-1241](https://jira.percona.com/browse/PSMDB-1241) - Implement the `connectRetries` and the `connectTimeoutMS` configuration file options
```

Comment:
PSMDB-1241 was not ported to v4.2 branch. This should be removed.
@ktrushin correct me if I'm wrong

Reply:
Fixed